src/rpc/core_rpc_server_commands_defs.h

```diff
@@ -176,6 +176,344 @@ namespace cryptonote
+          KV_SERIALIZE(coinbase)
+          KV_SERIALIZE(mempool)
+          KV_SERIALIZE(mixin)
+        END_KV_SERIALIZE_MAP()
+      };
+      
+      
+      struct response
+      {
+        //std::list<std::string> txs_as_json;
+        std::string total_received;
+        uint64_t scanned_height;
+        std::list<transaction> transactions;
+        uint64_t blockchain_height;
+        uint64_t scanned_block_height;
+        BEGIN_KV_SERIALIZE_MAP()
+          KV_SERIALIZE(total_received)
+          KV_SERIALIZE(scanned_height)
+          KV_SERIALIZE(transactions)
+          KV_SERIALIZE(blockchain_height)
+          KV_SERIALIZE(scanned_block_height)
+        END_KV_SERIALIZE_MAP()
+      };
+  };
+  
+  //-----------------------------------------------
+  struct COMMAND_RPC_LIGHT_WALLET_GET_ADDRESS_INFO
+  {
+      struct request
+      {
+        std::string address;
+        std::string view_key;
+
+        BEGIN_KV_SERIALIZE_MAP()
+          KV_SERIALIZE(address)
+          KV_SERIALIZE(view_key)
+        END_KV_SERIALIZE_MAP()
+      };
+      
+      struct spent_output 
+      {
+        std::string amount;
+        std::string key_image;
+        std::string tx_pub_key;
+        int out_index;
+        int mixin;
+        
+        
+        BEGIN_KV_SERIALIZE_MAP()
+          KV_SERIALIZE(amount)
+          KV_SERIALIZE(key_image)
+          KV_SERIALIZE(tx_pub_key)
+          KV_SERIALIZE(out_index)
+          KV_SERIALIZE(mixin)
+        END_KV_SERIALIZE_MAP()
+      };
+  
+      
+      
+      struct response
+      { 
+        std::string locked_funds;
+        std::string total_received;
+        std::string total_sent;
+        uint64_t scanned_height;
+        uint64_t scanned_block_height;
+        uint64_t start_height;
+        uint64_t transaction_height;
+        uint64_t blockchain_height;
+        std::list<spent_output> spent_outputs;
+        BEGIN_KV_SERIALIZE_MAP()
+          KV_SERIALIZE(locked_funds)
+          KV_SERIALIZE(total_received)
+          KV_SERIALIZE(total_sent)        
+          KV_SERIALIZE(scanned_height)
+          KV_SERIALIZE(scanned_block_height)
+          KV_SERIALIZE(start_height)
+          KV_SERIALIZE(transaction_height)
+          KV_SERIALIZE(blockchain_height)
+          KV_SERIALIZE(spent_outputs)
+        END_KV_SERIALIZE_MAP()
+      };
+  };
+  
+  //-----------------------------------------------
+  struct COMMAND_RPC_LIGHT_WALLET_GET_UNSPENT_OUTS
+  {
+      struct request
+      {
+        std::string amount;
+        std::string address;
+        std::string view_key;
+
+        BEGIN_KV_SERIALIZE_MAP()
+          KV_SERIALIZE(amount)
+          KV_SERIALIZE(address)
+          KV_SERIALIZE(view_key)
+        END_KV_SERIALIZE_MAP()
+      };
+    
+      
+      struct output {
+        std::string amount;
+        std::string public_key;
+        int index;
+        uint64_t global_index;
+        std::string rct;
+        uint64_t tx_id;
```

Comment:
Is this the lmdb internal thingie ? If so, can you find another way to get the info you want ? A new DB implementation might not have a concept of it. Otherwise, if it is an actual hash, this should not be working :)